### ISSUE TITLE
xtest FakeInput logs undefined and <Buffer 00 00 ... 00 >

### ISSUE THREAD

**Issue by dizzib** [sidorares/node-x11#83]
Is there any way to prevent it writing entries everytime I press a key ? Ta.

    [andy@sandpit smoketest]$ node xtesttest.js 
    { present: 1,
      majorOpcode: 132,
      firstEvent: 0,
      firstError: 0,
      GetVersion: [Function],
      KeyPress: 2,
      KeyRelease: 3,
      ButtonPress: 4,
      ButtonRelease: 5,
      MotionNotify: 6,
      FakeInput: [Function] }
    click
    undefined
    <Buffer 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00>
    undefined
    <Buffer 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00>
     click
     click
     undefined
    <Buffer 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00>
    undefined
    <Buffer 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00>
    ^C[andy@sandpit smoketest]$

**Comment by santigimeno** [comment]
With latest master I'm not seeing any Buffer logs. My output is just this whether I press a key or not:

```
node examples/smoketest/xtesttest.js 
{ present: 1,
  majorOpcode: 132,
  firstEvent: 0,
  firstError: 0,
  GetVersion: [Function],
  KeyPress: 2,
  KeyRelease: 3,
  ButtonPress: 4,
  ButtonRelease: 5,
  MotionNotify: 6,
  FakeInput: [Function] }
click
click
click
click
click
click
click
```

**Comment by sidorares** [comment]
yes, there used to be `console.log` hanging around at the end of event parser but not in master

**Comment by sidorares** [comment]
let us know the version you have. If that's latest npm I might publish update

**Comment by dizzib** [comment]
v1.0.2 from npm

**Comment by sidorares** [comment]
thanks for report @dizzib  - published master as v1.0.3 !

**Comment by dizzib** [comment]
super, thanks! :)
